winpr/libwinpr/sysinfo/sysinfo.c

```diff
@@ -678,6 +678,21 @@ BOOL IsProcessorFeaturePresent(DWORD ProcessorFeature)
 
 #endif //_WIN32
 
+DWORD GetTickCountPrecise(void)
+{
+#ifdef _WIN32
+	LARGE_INTEGER freq;
+	LARGE_INTEGER current;
+
+	QueryPerformanceFrequency(&freq);
+	QueryPerformanceCounter(&current);
+
+	return (DWORD) (current.QuadPart * 1000LL / freq.QuadPart);
+#else
+	return GetTickCount();
```

Comment:
On Linux, GetTickCount() is implemented using clock_gettime(CLOCK_MONOTONIC) which I believe is already high-precision, so we just have to wrap the new API to it directly. It is only on Windows that GetTickCount() has such problem.